openassessment/xblock/openassessmentblock.py

```diff
@@ -588,7 +588,8 @@ def ora_blocks_listing_view(self, context=None):
             additional_css=["static/css/lib/backgrid/backgrid%s.css" % min_postfix],
             additional_js=["static/js/lib/backgrid/backgrid%s.js" % min_postfix],
             additional_js_context={
-                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled
+                "ENHANCED_STAFF_GRADER": self.is_enhanced_staff_grader_enabled,
+                "ORA_GRADING_MICROFRONTEND_URL": settings.ORA_GRADING_MICROFRONTEND_URL
```

Comment:
This line was lost during rebase which is why the ora grading link was incorrect.